cms/toolbar/toolbar.py

```diff
@@ -176,7 +178,7 @@ def get_menu(self, key, verbose_name=None, side=LEFT, position=None):
             return self.menus[key]
         return None
 
-    def get_or_create_menu(self, key, verbose_name=None, side=LEFT, position=None):
+    def get_or_create_menu(self, key, verbose_name=None, disabled=False, side=LEFT, position=None):
```

Comment:
I thought it'd best to keep a consistent signature on related methods. I'll add a note in the release notes.